flask_lesson_intro/templates/headphones.html

```diff
@@ -0,0 +1,5 @@
+{% extends "base_product.html" %}
+{% block title %}{{title}}{% endblock %}
+{% block image %}{{url_for('static', filename='headphones.jpg')}}{% endblock %}
+{% block text %}{{text}}{% endblock %}
+{% block count %}{{count}}{% endblock %}
```

Comment:
you do not need all this templates and routes, try to create one route with one template for all cases